### DISCUSSION TITLE
GitHub图床证书过期

### DISCUSSION THREAD

**Discussion by MCheping8108** [community/community#33772] [Other Feature Feedback, Questions, & Ideas]
![Snipaste_2022-09-22_08-30-00](https://user-images.githubusercontent.com/102772286/191633634-942771e3-a4bf-4a68-ae35-30e32d1823cb.png)

**Comment by Zyilin98** [comment]
![image](https://user-images.githubusercontent.com/61766598/191638470-0ee696d5-e077-412e-a212-51fcae44d06c.png)
![image](https://user-images.githubusercontent.com/61766598/191638741-3779d077-fd13-44e5-b15b-9ee26a4a65a9.png)

咱就是说你有没有可能被mitm了

**Reply by MCheping8108** [reply]
震惊.jpg
我怎么不知道？

**Reply by Zyilin98** [reply]
https://developer.mozilla.org/zh-CN/docs/Glossary/MitM
你设备的问题 自己去查
这服务要是真的证书过期了 就不是你一个人反馈了 这事情可大了
~~我都觉得丢人~~

**Reply by MCheping8108** [reply]
怎么解决？

**Reply by MCheping8108** [reply]
草，要不我还是删除吧
